vllm/lora/layers.py

```diff
@@ -1067,6 +1067,10 @@ def scale(self):
     def soft_cap(self):
         return self.base_layer.soft_cap
 
+    @property
+    def use_gather(self):
+        return self.base_layer.use_gather
+
```

Comment:
what is this used for?

Reply:
This class wraps the logits processor class and requires exposing every property under `self` this way. Without this, we'll see failures in lora tests.